docs/tutorials/google/echoes.ipynb

```diff
@@ -0,0 +1,686 @@
+      "text/plain": [
+       "<Figure size 504x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light",
+      "tags": []
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Average data over trials.\n",
+    "avg_probs = np.average(probs, axis=0)\n",
+    "std_probs = np.std(probs, axis=0)\n",
+    "\n",
+    "# Plotting.\n",
+    "plt.figure(figsize=(7, 5))\n",
+    "\n",
+    "step = len(cycle_values)\n",
+    "stop = len(avg_probs) // step\n",
+    "for i in range(stop):\n",
+    "    plt.errorbar(\n",
+    "        x=cycle_values, \n",
+    "        y=avg_probs[i * step: (i + 1) * step], \n",
+    "        yerr=std_probs[i * step: (i + 1) * step],\n",
+    "        capsize=5,\n",
+    "        lw=2,\n",
+    "        label=f\"Qubit configuration {i}\"\n",
+    "    )\n",
+    "\n",
+    "plt.legend()\n",
+    "plt.ylabel(\"Survival probability\")\n",
+    "plt.xlabel(\"Cycle\")\n",
+    "plt.grid(\"on\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "0WGZ9T2q6YI1"
+   },
+   "source": [
+    "The initial point (at zero cycles) reflects readout error, and the decay rate reflects the gate error per cycle. We fit an exponential to each curve to determine the gate error per cycle below."
```

Comment:
[No change needed] This plot is really cool! I can immediately see how the higher error in qubit set 1 corresponds to a faster survival decay rate.